lms/static/sass/course/_auto-cert.scss

```diff
@@ -0,0 +1,42 @@
+.progress-certificates {
+    margin-bottom: 0 !important;
```

Comment:
No importants please. There is already a rule scoped to only this case on line 136 of this file: 

edx-platform/lms/static/sass/course/_profile.scss

so I think you are safe just editing that rule. But in general, it's better to question the request than throw importants in.